mmpy_bot/function.py

```diff
@@ -80,6 +80,8 @@ def __init__(
         else:
             self.allowed_channels = [channel.lower() for channel in allowed_channels]
 
+        self.docstring = self.function.__doc__ or ""
```

Comment:
Why can this not be defined on the parent class?

Reply:
Done